src/apiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getAllMovies = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(res => res.json())
+}
+
+export const getSingleMovie = (id) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+  .then(res => res.json()) 
+}
```

Comment:
I really like how you separated the file for only API, I'm happy that this worked! 🙂